rest/importtest/collections_import_test.go

```diff
@@ -136,4 +146,100 @@ func TestMultiCollectionImportFilter(t *testing.T) {
 		assertDocProperty(t, response, "id", "TestImportFilterInvalid")
 		assertDocProperty(t, response, "rev", "1-25c26cdf9d7771e07f00be1d13f7fb7c")
 	}
+	// Add a collection
+	scopesConfig = rest.GetCollectionsConfig(t, testBucket, 3)
+	dataStoreNames = rest.GetDataStoreNamesFromScopesConfig(scopesConfig)
+
+	scopesConfig[dataStoreNames[0].ScopeName()].Collections[dataStoreNames[0].CollectionName()] = rest.CollectionConfig{ImportFilter: &importFilter1}
+	scopesConfig[dataStoreNames[1].ScopeName()].Collections[dataStoreNames[1].CollectionName()] = rest.CollectionConfig{ImportFilter: &importFilter2}
+	scopesConfig[dataStoreNames[2].ScopeName()].Collections[dataStoreNames[2].CollectionName()] = rest.CollectionConfig{ImportFilter: &importFilter3}
+
+	scopesConfigString, err := json.Marshal(scopesConfig)
+	require.NoError(t, err)
+
+	response = rt.SendAdminRequest("PUT", "/db/_config", fmt.Sprintf(
+		`{"bucket": "%s", "num_index_replicas": 0, "enable_shared_bucket_access": %t, "scopes":%s}`,
+		testBucket.GetName(), base.TestUseXattrs(), string(scopesConfigString)))
+	rest.RequireStatus(t, response, http.StatusCreated)
+	rt2 := rest.NewRestTesterMultipleCollections(t, rtConfig, 3)
```

Comment:
Initializing another RestTester in general means this test requires two buckets to run, but it doesn't here because `RestTester` is constructed with a non closeable bucket. This is not quite the configuration we want to necessarily be testing because like two SG running that are targeting the same bucket (the first hasn't shut down yet).

In Jenkins, this doesn't matter, but in dev running tests under GSI, I use `SG_TEST_BUCKET_POOL_SIZE=1` and `SG_TEST_COLLECTION_POOL_SIZE=1` to speed up the test harness. This only works when you have set this up. I'd match `base.RequireNumTestDataStores(t, 3)` as a constant with `numCollections` you create. I'd rather matching maxNumCollections here.

Reply:
[1df3251](https://github.com/couchbase/sync_gateway/pull/6030/commits/1df32518eb2600aa65b659ff1fb1f5c07c334d2e)